src/engine/controllers/projectController.ts

```diff
@@ -97,14 +98,18 @@ export class ProjectController {
     const name = path.basename(context.config.rootExecutionDir);
     context.logger.debug('start initialize project "' + name + '"');
 
-    context.logger.debug('load main yml file');
-    const config = ProjectController.loadConfigFile(context);
-
-    context.logger.debug('load extensions');
-    const extensions = ProjectController.loadExtensions(config);
-
-    const gqlSchema = GraphqlController.loadSchema(ProjectController.getSchemaPaths(extensions));
-
+    const projectFile = ProjectController.checkProjectFileExist(context);
+    const extensions = projectFile.extensions;
+    const gqlSchema = projectFile.gqlSchema;
+
+    context.logger.debug('initialize plugins structure');
+    const pluginPaths = this.loadConfigFile(context).plugins;
+    if (pluginPaths) {
+      pluginPaths.map((plugin: { path: string }) => {
+        const pluginDirname = path.dirname(path.join(context.config.rootExecutionDir, plugin.path));
+        ProjectController.checkProjectFileExist(context, pluginDirname);
+      });
+    }
```

Comment:
I think we need to transform `checkProjectFileExist` into `validateProjectFile`

```suggestion
    context.logger.debug('initialize plugins structure');
    const { plugins } = this.loadConfigFile(context);
    
    if (plugins) {
      plugins.map((plugin) => {
        const pluginDir = path.dirname(path.join(context.config.rootExecutionDir, plugin.path));
        ProjectController.checkProjectFileExist(context, pluginDir);
      });
    }
```